FordLabs/retroquest
# Add application version tab to settings gear

Co-authored-by: Nick Reuter <thenewimagineer@gmail.com>
Co-authored-by: Ashley Clifton <randomashes@gmail.com>

## Overview
Adds a tab to the setting gear to display the commit hash of the running application. Should help identify a problem with a running application.

### Demo
<img width="1026" alt="Screen Shot 2022-01-14 at 2 56 39 PM" src="https://user-images.githubusercontent.com/37785108/149577132-26886ab8-bf47-4481-bdd1-09f3d61d64ee.png">

### Notes
We should add further tracing to be able to replay running retros and easily reproduce issues locally, but that can be dealt with later.

## Testing Instructions
* Build the application
* Run the application from the backend
* Create a board and login
* Click the settings gear and navigate to the info tab
* verify that the value is the same as the short-hash of the commit
